.github/workflows/deploy.yml

```diff
@@ -87,10 +87,10 @@ jobs:
         DEPLOY_ENV: nonprod
         DATALAKE_SAML_IDENTITY_PROVIDER_ARN: ${{ secrets.DATALAKE_SAML_IDENTITY_PROVIDER_ARN_NON_PROD }}
 
-    # PROD DEPLOYMENT
+    # PROD DEPLOYMENT - in release branch, deploy on tag, otherwise report stack changes only
     - name: (Prod) Configure AWS credentials
       if: >
-        startsWith(github.ref, 'release')
+        startsWith(github.ref, 'refs/heads/release-') || startsWith(github.ref, 'refs/tags/release-')
```

Comment:
It would be simpler if we only deploy on tags, rather than allowing either. What do you think?

Reply:
It is only setting of credentials. I allow both actions  - push to branch and tag because if action is only push to release branch we report `cdk diff` with current prod deployment. Production deployment happens only if tag is pushed. But credentials are needed for both cases.